src/editors/CommonAssignmentTypeEditor.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {handleFormUpdate} from '../utils';
+
+/**
+ * This component contains the form fields common to all graph types,
+ * like title, description, etc.
+ */
+export default class CommonGraphEditor extends React.Component {
```

Comment:
This needs to be CommonAssignmentTypeEditor here, instead of CommonGraphEditor.

Assignment Type is really just one attribute of the graph, though: the assignment type. I don't think it makes sense to include other attributes here. Instead, I think this should be renamed to something like Assignment Settings. Then again, some of the settings here, like "Display intersection", are useful even if this isn't an assignment, so maybe just Settings or Graph Settings.

Reply:
Agreed re the name issue - I changed it to `CommenGraphSettings`. I also updated the class name where needed.